GLS edit of Priority Sliders completed (#99)
rht-labs/tech-exercise: docs/slides/content/priority-sliders.md

<!-- .slide: data-background-size="contain" data-background-image="images/priority-sliders/blank-sliders.png", data-background-opacity="0.1" -->

### Exercise - Priority Sliders
In this exercise you will create a priortiy slider canvas for Pet Battle. 
#### *In your team*
1. Choose roles for the team members to role play: 
   * Head of Architecture
   * Head of Security
   * Head of Business
   * Everyone else is a cross-functional team member

### Exercise - Priority Sliders
#### *In your team*
2. Define the topics that are relevant for the Pet Battle use case and place those topics in the row headings. 
3. Individually write down the priority number for each heading
4. Try to come to consensus by sliding the priorities against one another.

# Exercise Wrap Up
Q & A / Feedback

How did you reach consensus?
![what-is-it](images/priority-sliders/consensus-tools.png)<!-- .element: class="image-no-shadow image-full-width" -->

#### Facilitation Tips

> * Topics are not product features but product area are team-building topics.

Reviewer: ```suggestion
* Topics are not product features but product area or team-building topics.
```

#### Facilitation Tips
* Topics are not product features but product area are team-building topics.

> <!--Tech Editor: I don't understand this first bullet, and "product area are team building topics" has a subject-verb agreement error and is unclear. The first part is ok, "topics are not product features", but the second part confused me. Please edit for better clarity. -->

Reviewer: just asked for clarification. I'll update here later on :)

Reviewer 2: A quick scan but I think it was meant to say `Topics are not product features but product area or team-building topics`

Reviewer: ```suggestion
```

Thanks @springdo :) @nmuller66 let's update that sentence as Donal shared and remove the comment - then we can merge it :)

> * Bring to your Sprint Planning: can be used as a filtering mechanism for product backlog items prioritization.

Author: ```suggestion
* Bring priority slider artifacts to sprint planning to use when prioritizing items in the backlog.
```

> <!--TechEdit: Does this mean "Bring priority slider artifacts to sprint planning to use when prioritizing items in the backlog." Please update. -->

Reviewer: exactly like this 🙈 happy to use your comment!

> <!--TechEdit: Does this mean "Bring priority slider artifacts to sprint planning to use when prioritizing items in the backlog." Please update. -->

Author: ```suggestion

```
